web/pdf_page_view.js

```diff
@@ -979,6 +979,7 @@ class PDFPageView {
             pdfPage,
             l10n,
             accessibilityManager: this._accessibilityManager,
+            annotationLayer: this.annotationLayer.annotationLayer,
```

Comment:
Depending on the options used the `annotationLayer` may not be available, hence:
```suggestion
            annotationLayer: this.annotationLayer?.annotationLayer,
```